content/en/docs/started/workstation/getting-started-multipass.md

```diff
@@ -7,21 +7,15 @@ weight = 60
 {{% alpha-status 
   feedbacklink="https://github.com/kubeflow/kubeflow/issues" %}}
 
-This document outlines the steps that you can take to get your local installation of Kubeflow running on top of Microk8s, a small enterprise Kubernetes cluster. Microk8s requires Linux; if you are not on a Linux system, you can use Multipass to create a Linux VM (virtual machine) on your native hypervisor.
 
 ## Introduction
 
-If you already have Ubuntu or Linux that supports [snaps](https://snapcraft.io/), you can easily install Kubernetes using [Microk8s](https://microk8s.io/). You can jump to the **Install Kubeflow using Microk8s** section below.
+This document outlines the steps that you can take to get your local installation of Kubeflow running on top of [Microk8s](https://microk8s.io/), a small enterprise Kubernetes cluster. Microk8s is now available on Windows, MacOs and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website. 
```

Comment:
I propose to simplify your first sentence by cutting some words out, while retaining the meaning. At the moment, it's a bit verbose: "This doc outlines the steps that you can take to..."

```diff
This 
- document outlines the steps that you can take 
+ guide will show you how
- to get your local 
+ to deploy Kubeflow
- installation of Kubeflow 
- running on top of 
+ on [Microk8s](https://microk8s.io/)
- , a small enterprise Kubernetes cluster.
+ —a small enterprise Kubernetes cluster.
```

Reply:
Awesome! Thank you for this review 🚀